packages/io/x3d-serializer/src/index.js

```diff
@@ -202,7 +205,7 @@ const convertMesh = (object, options) => {
 
   const faceset = [
     'IndexedTriangleSet',
-    { ccw: 'true', colorPerVertex: 'false', solid: 'false', index: indexList },
+    { ccw: 'true', colorPerVertex: 'false', normalPerVertex: options.smooth === true, solid: 'false', index: indexList },
```

Comment:
Maybe options.smooth is enough. Typically, the values of options are not checked but... correct values are expected.

Reply:
will change to options.smooth .